mobile/.env

```diff
@@ -0,0 +1,8 @@
+#Front end
```

Comment:
I'd recommend adding .env to gitignore and not committing it. Similar to Telescope let's make `example.env` the only env file which makes it into project.

Reply:
ah, yes